backend/src/main/resources/application.yaml

```diff
@@ -47,6 +47,8 @@ spring:
           keycloak-moh:
             client-id: USER-MANAGEMENT-SERVICE
             client-secret: ${UMS_CLIENT_SECRET}
+            client-test-id: USER-MANAGEMENT-SERVICE-INTEGRATION-TESTS
+            client-test-secret: ${UMS_CLIENT_TEST_SECRET}
```

Comment:
It should be possible to put these properties in a test configuration file.

https://github.com/bcgov/moh-organizations-api/tree/master/src/test/resources

We do not want to complicate our configuration by mixing production and test properties.